src/services/codefixes/importFixes.ts

```diff
@@ -894,7 +894,7 @@ function getSingleExportInfoForSymbol(symbol: Symbol, symbolName: string, module
 }
 
 function isFutureSymbolExportInfoArray(info: readonly SymbolExportInfo[] | readonly FutureSymbolExportInfo[]): info is readonly FutureSymbolExportInfo[] {
-    return info[0].symbol === undefined;
+    return info[0].moduleSymbol === undefined;

```

Comment:
This looks incorrect. `moduleSymbol` is a required non-nullable property in `FutureSymbolExportInfo`; `symbol` must be `undefined`